python/packages/autogen-ext/pyproject.toml

```diff
@@ -48,10 +48,13 @@ video-surfer = [
     "ffmpeg-python",
     "openai-whisper",
 ]
-
 grpc = [
     "grpcio~=1.62.0", # TODO: update this once we have a stable version.
 ]
+jupyter-executor = [
+    "ipykernel>=6.29.5",
+    "jupyter-kernel-gateway>=3.0.1",
+]
```

Comment:
Looks like you need to update the dependencies for this new version

Reply:
Done